main/src/library/Path.flix

```diff
@@ -126,6 +127,41 @@ namespace Path {
         javaFile.canWrite()
     })
 
+    ///
+    ///
+    ///
+    def mapInputStreamError(err: Core/Io/InputStream.Error): IOError = {
+        use Core/Io/InputStream.Error;
+
+        match err {
+            case Error.IOException => Generic("I/O Error in InputStream")
+            case Error.IndexOutOfBoundsException => Generic("Index Out Of Bounds in InputStream")
+        }
+    }
+
+    ///
+    /// Read the specified path into an array of bytes
+    ///
+    pub def readBuffer(p: Path): Result[Array[Int8], IOError] & Impure = try {
```

Comment:
I wonder if this belongs on `Path` or in some other namespace.

Reply:
I could move it to `Core/Io/InputStream` and make it something like `fileInputStream(p: Path): Result[InputStream, IOError]`. This would also enable the consumer to further choose to wrap the `FileInputStream` in a `BufferedInputStream` if they so choose.

Reply:
Let us leave it here for now.